docker-compose-services/drupal8-behat-selenium/docker-compose.selenium.yaml

```diff
@@ -5,9 +5,6 @@ services:
     depends_on:
     - db
     - selenium
-    links:
-    - db:db
-    - selenium:selenium
```

Comment:
Are you sure the selenium link is not required? I would bet it's not required. It would just make sure that the selenium container is accessible as "selenium" from the web container... but the web container doesn't need access to the selenium container right? Just vice versa?